benchmark/src/ckks-bootstrapping-precision.cpp

```diff
@@ -0,0 +1,140 @@
+//==================================================================================
```

Comment:
It looks like an example rather than a Google benchmark file. I suggest moving this files to extras for pke.

Reply:
Moved.